src/Cake.FileHelpers.Tests/Cake.FileHelpers.Tests.csproj

```
@@ -1,26 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net50;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+    <PackageReference Include="Cake.Testing" Version="1.0.0"  />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net461' ">
+    <PackageReference Include="Cake.Testing" Version="2.0.0" />
+  </ItemGroup>

```

```suggestion
  <ItemGroup>
    <PackageReference Include="Cake.Testing" Version="2.0.0" />
  </ItemGroup>
```

Those are not needed, as `net461` is no longer needed.

---

done